### DISCUSSION TITLE
Help me with circuit termination to a rear port

### DISCUSSION THREAD

**Discussion by dejantep** [netbox-community/netbox#13739] [Help Wanted!] [answered]
I trying to document a circuit between two CWDM devices. Devices are modelled with front/rear ports. Front ports are different wavelenghts and rear port is so called network port. This is what i want to achieve

`cwdm device 1 rear port` - `circuit term A` - `circuit term Z` - `rear port cwdm device 2`

If i'm not missing something this is not possible but to me this is correct representation of reality. We are renting dark fiber between cwdm devices.
Do i need to change haow i modell cwdm devices or is there another way?

**Comment by candlerb** [comment, marked as answer]
AFAIK it *should* be possible to create a cable between a rear port and a circuit termination. Have you tried this, and if so, what error do you get?

OTOH, tracing through this almost certainly won't work (at least, not if you want to trace all the wavelengths individually).

Your other option would be to make a cable directly between device 1 rear port and device 2 rear port.  There's nothing in Netbox to prevent a cable being cross-site.  But then it won't be visible as a circuit.

Netbox has a number of limitations in its cabling model. There's no explicit modelling of DWDM, and there's no way to carry multiple provider circuits over a single bearer (whether that be as VLANs or as wavelengths)

**Comment by dejantep** [comment]
You are right. It is possible to connect rear port to a circuit termination. I was thrown off by edit function of direct cable that was there. It has to be removed to be able to make new connection. Of course.

You are also right regarding tracing. It doesnt work at all.

At least now i have circuit documented as it should be
